jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
some additional config necessary specific to sonar  jest-sonar-reporter .. can be done as part of typescript migration phase 2

see details on [defects commit ](https://github.com/dvsa/cvs-svc-defects/commit/42bddbc8803d0965d950cc8fdb5bc694db16bd70)